Content.Shared/_RMC14/Atmos/RMCFireImmunityBypassComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Shared._RMC14.Atmos;
+
+/// <summary>
+/// Component that allows fires to bypass fire immunity.
+/// This should be added to fire entities (like certain tile fires) to make them
+/// able to damage entities that normally have fire immunity.
+/// </summary>
+[RegisterComponent]
+public sealed partial class RMCFireImmunityBypassComponent : Component;
```

Comment:
Missing NetworkedComponent for a Shared component

```suggestion
using Robust.Shared.GameStates;

namespace Content.Shared._RMC14.Atmos;

/// <summary>
/// Component that allows fires to bypass fire immunity.
/// This should be added to fire entities (like certain tile fires) to make them
/// able to damage entities that normally have fire immunity.
/// </summary>
[RegisterComponent, NetworkedComponent]
public sealed partial class RMCFireImmunityBypassComponent : Component;
```